contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -3407,7 +3413,10 @@ bbf_ProcessUtility(PlannedStmt *pstmt,
 									}
 
 									pfree(rolspec->rolename);
+									pfree(db_owner_name);
+
 									rolspec->rolename = user_name;
+									pfree(user_name);
```

Comment:
We are freeing something we are still using.

Reply:
Done